appveyor.yml

```diff
@@ -1,70 +1,128 @@
-  - IF %CONFIGURATION%==Release sed -e "s/{VERSION}/%XWORD_VERSION%/" -e "s/{OS_FILE}/XWord-Windows.exe/" dist/packages_template.lua > dist/packages_windows.lua
-
-# Zip the full build folder and make it available as an artifact.
-after_build:
-  - 7z a bin\%CONFIGURATION%\XWord-Windows.zip "%APPVEYOR_BUILD_FOLDER%\bin\%CONFIGURATION%\*" -r -x!*.exp -x!*.lib -x!*.pdb -x!*.idb -x!*.ilk -x!*.fbp
-
-artifacts:
-  - path: bin\%CONFIGURATION%\XWord-Windows.zip
-    name: XWord-Windows.zip
-
-  - path: dist\XWord-Windows.exe
-    name: XWord-Windows.exe
-
-  - path: dist\packages_windows.lua
-    name: packages_windows.lua
+for:
+-
+  # Windows
+  matrix:
+    only:
+      - image: Visual Studio 2015
+
+  environment:
+    CURL_VERSION: 7.69.0
+    PYTHON: "C:\\Python38"
+
+  install:
+    # For tagged builds, use the tag name as the version.
+    # Otherwise, read the version from the VERSION file.
+    - IF "%APPVEYOR_REPO_TAG%"=="true" (SET XWORD_VERSION=%APPVEYOR_REPO_TAG_NAME%) ELSE (SET /P XWORD_VERSION=< VERSION)
+    # Build and install markdown
+    - SET PATH=%PYTHON%;%PATH%
+    - IF %CONFIGURATION%==Release python -m pip install markdown
+    - call "%ProgramFiles(x86)%\Microsoft Visual Studio 12.0\VC\vcvarsall.bat"
+    # Build curl
+    - bash build-curl.sh %CURL_VERSION%
+    # Build LuaJIT
+    - bash build-luajit.sh %LUAJIT_VERSION%
+    # Build wxWidgets, with our custom tweaks patch
+    - bash build-wxwidgets.sh %WX_VERSION% %CONFIGURATION%
+
+  # Cache the compiled libraries so they can be used with future builds
+  cache:
+    - '%USERPROFILE%\wxWidgets-%WX_VERSION%'
+    - '%USERPROFILE%\curl-%CURL_VERSION%'
+    - '%USERPROFILE%\LuaJIT-%LUAJIT_VERSION%'
+
+  # Run premake to create the Visual Studio project
+  before_build:
+    - premake5.exe --wx-prefix="%USERPROFILE%/wxWidgets-%WX_VERSION%" vs2013
+
+  # Build XWord
+  build_script:
+    - msbuild build/vs2013/XWord.sln /p:Configuration=%CONFIGURATION%
+    # For release builds, also build docs and run dist steps.
+    - IF %CONFIGURATION%==Release pushd doc & make_help.cmd & popd
+    - IF %CONFIGURATION%==Release pushd dist & python gen_nsis.py > scripts.nsi & "%ProgramFiles(x86)%\NSIS\makensis.exe" /V4 XWord.nsi & popd
+    - IF %CONFIGURATION%==Release sed -e "s/{VERSION}/%XWORD_VERSION%/" -e "s/{OS_FILE}/XWord-Windows.exe/" dist/packages_template.lua > dist/packages_windows.lua
+
+  # Zip the full build folder and make it available as an artifact.
+  after_build:
+    - 7z a bin\%CONFIGURATION%\XWord-Windows.zip "%APPVEYOR_BUILD_FOLDER%\bin\%CONFIGURATION%\*" -r -x!*.exp -x!*.lib -x!*.pdb -x!*.idb -x!*.ilk -x!*.fbp
+
+  artifacts:
+    - path: bin\%CONFIGURATION%\XWord-Windows.zip
+      name: XWord-Windows.zip
+
+    - path: dist\XWord-Windows.exe
+      name: XWord-Windows.exe
+
+    - path: dist\packages_windows.lua
+      name: packages_windows.lua
+
+-
+  # Mac
+  matrix:
+    only:
+      - image: macOS
+
+  environment:
+    PYTHON: "C:\\Python38"
```

Comment:
I'm guessing we don't need this?

Reply:
Whoops, good catch! No, we don't - Python is only used to build the Windows Help files + documentation.